schema.graphql

```diff
@@ -22,3 +22,31 @@ type Payment @entity {
   feeAmountInCrypto: BigDecimal
   maxRateTimespan: Int
 }
+
+type EscrowEvent @entity {
+  id: ID!
+  contractAddress: Bytes!
+  reference: Bytes!
+  block: Int!
+  timestamp: Int!
+  txHash: Bytes!
+  eventType: String!
```

Comment:
maybe use an enum here
example from the doc:
```
enum TokenStatus {
  OriginalOwner
  SecondOwner
  ThirdOwner
}
```

Reply:
Yes, good idea